scripts/update-known-events.js

```diff
@@ -3,6 +3,9 @@ const Table = require("table-builder");
 const WEBOOOKS = require("@octokit/webhooks-definitions/index.json");
 const { generateFile } = require("./generate-file");
 
+const doNotEditThisFileDisclaimer = `
+// DO NOT EDIT THIS FILE DIRECTLY
```

Comment:
I think I would slightly rephrase it
```suggestion
// THIS FILE IS GENERATED - DO NOT EDIT DIRECTLY
```